src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -10,31 +9,31 @@ import SelectionStudy from "./SelectionStudy";
 import FileLookup from './FileLookup';
 import Error from "../../shared/error";
 import { PageSection, PageSectionVariants } from "@patternfly/react-core";
+import { DcmImage } from "../../context/reducers/dicomImagesReducer";
 
 interface CreateAnalysisDetailProps {
   setIsExpanded: Dispatch<SetStateAction<boolean>>,
   submitAnalysis: () => void
 }
 
 const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = ({setIsExpanded, submitAnalysis}) => {
-  const { state: { dcmImages, createAnalysis }, dispatch } = useContext(AppContext);
-  const { selectedStudyUIDs } = createAnalysis;
+  const { state: { createAnalysis, dcmImages }, dispatch } = useContext(AppContext);
   const [isXray, setIsXray] = useState(false);
+  const [patientName, setPatientName] = useState('');
+  const [patientBirthdate, setPatientBirthdate] = useState('');
+  const [patientSex, setPatientSex] = useState('');
 
   useEffect(() => {
-    const patientInfo = CreateAnalysisService.extractPatientPersonalInfo(dcmImages?.allDcmImages[0])
```

Comment:
Why do we no longer want to keep the patient info in the store?

Reply:
The info was only being used in one place, and it cleans up the code quite a lot to simply store the values inside the state and then display those accordingly, rather than storing into store, then immediately using it once then ignoring it